tests/providers/gcp/gcp_test.py

```diff
@@ -180,6 +182,87 @@
     }]
 }
 
+MOCK_STACKDRIVER_METRIC = 6693417
+MOCK_COMPUTE_METRIC = 8093
+MOCK_LOGGING_METRIC = 1
+
+MOCK_GCM_METRICS_COUNT = {
+    'timeSeries': [{
+        'metric': {
+            'type': 'serviceruntime.googleapis.com/api/request_count'
+        },
+        'resource': {
+            'type': 'consumed_api',
+            'labels': {
+                'project_id': 'fake-target-project',
+                'service': 'stackdriver.googleapis.com'
+            }
+        },
+        'metricKind':
+            'DELTA',
+        'valueType':
+            'INT64',
+        'points': [{
+            'interval': {
+                'startTime': '2020-05-18T00:00:00Z',
+                'endTime': '2020-06-17T00:00:00Z'
+            },
+            'value': {
+                'int64Value': MOCK_STACKDRIVER_METRIC
+            }
+        }]
+    }, {
+        'metric': {
+            'type': 'serviceruntime.googleapis.com/api/request_count'
+        },
+        'resource': {
+            'type': 'consumed_api',
+            'labels': {
+                'service': 'compute.googleapis.com',
+                'project_id': 'fake-target-project'
+            }
+        },
+        'metricKind':
+            'DELTA',
+        'valueType':
+            'INT64',
```

Comment:
I know this is probably YAPF's fault, but it would be more readable is these were on the same line. Same comments for other sections of the JSON blob that are the same

Reply:
Fixed